src/main/java/net/masterthought/cucumber/json/support/TagObject.java

```diff
@@ -153,9 +153,4 @@ public Status getStatus() {
     public String getRawStatus() {
         return status.name().toLowerCase();
     }
-
-    @Override
-    public String getDeviceName() {
```

Comment:
I don't care any more about how the report file is named as this may be confused for the report